### DISCUSSION TITLE
Uppercasing http verb macros

### DISCUSSION THREAD

**Discussion by pepe** [swlkr/osprey#2] [Ideas]
I just created #1 with WIP changes to http verbs. I think even in this half-baked early version, it looks better, the dreaded -in hack notwithstanding, and conveys the meaning of the functionality clearer. 

Feel free to challenge me!

Thanks for all your work with Janet!

**Comment by swlkr** [comment]
Haha no I think it's better with the upper case and without the `-in` everywhere!
